apps/web/modules/event-types/views/event-types-listing-view.tsx

```diff
@@ -911,6 +914,18 @@ const InfiniteScrollMain = ({
   const activeEventTypeGroup =
     eventTypeGroups.filter((item) => item.teamId === data.teamId) ?? eventTypeGroups[0];
 
+  const bookerUrl = orgBranding ? orgBranding?.fullDomain : WEBSITE_URL;
+
+  // If the event type group is the same as the org branding team, or the parent team, set the bookerUrl to the org branding URL
+  // This is to ensure that the bookerUrl is always the same as the one in the org branding settings
+  // This keeps the app working for personal event types that were not migrated to the org (rare)
+  if (
+    activeEventTypeGroup[0].teamId === orgBranding?.teamId ||
+    activeEventTypeGroup[0].parentId === orgBranding?.teamId
+  ) {
+    activeEventTypeGroup.bookerUrl = bookerUrl;
```

Comment:
@hariombalhara we should test this with embeds but from what i can see this works as expected